source/_docs/site-owner-faq.md

```diff
@@ -87,7 +87,7 @@ We do not recommend entering credit card information on behalf of a new owner. I
 
 Enterprise Organizations can use the same process to assume ownership of a site; however, Agency Partners do not have the ability to own sites directly.
 
-For Sandbox sites, the site owner can click **Team**, then click **Make Owner** next to the team member who should receive ownership of the site.
+For Sandbox sites, the site owner can click **Team**, then click **Make Owner** next to the site team member who should receive ownership of the site.
```

Comment:
Because this is stated in the header and in the beginning of the sentence, I recommend leaving it as "team member".